tests/integration_tests/test_login.py

```diff
@@ -0,0 +1,64 @@
+from flask import url_for
+from selenium.webdriver.support.ui import Select
+
+from tests import DEFAULT_PASSWORD, TEST_USERNAME
+from tests.integration_tests import IntegrationTestCase
+
+
+class TestLogin(IntegrationTestCase):
+    """Test class for Login integration tests"""
+
+    def test_login_page(self):
+        """Ensure login works properly"""
+        driver = self.driver
+        driver.get(url_for("user.login", _external=True))
+
+        driver.find_element_by_name("email").click()
+        driver.find_element_by_name("email").clear()
+        driver.find_element_by_name("email").send_keys(TEST_USERNAME)
+        driver.find_element_by_name("password").click()
+        driver.find_element_by_name("password").clear()
+        driver.find_element_by_name("password").send_keys(DEFAULT_PASSWORD)
+        driver.find_element_by_xpath(
+            "//input[@class='btn btn-tnth-primary btn-lg' and @value='LOG IN']").click()
+        driver.find_element_by_id("tnthUserBtn").click()
+        driver.find_element_by_link_text("Log Out of TrueNTH").click()
+
+    def test_consent_after_login(self):
```

Comment:
Can you add this test in a subsequent PR, or set as an [expected failure with pytest](https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail)?

It seems to have [broken every `ui` job](https://travis-ci.org/uwcirg/true_nth_usa_portal/jobs/456642263) since you added. NB: you'll have to open each TravisCI build to see since the `ui` "job" is allowed to fail in TravisCI

Reply:
I've been looking at them and I've been trying to fix it. I think I know why it's failing. At some point through the consent process a dialogue pops up and I don't think selenium likes it. I think there is a consent workflow that I'm hoping gets this test to pass. I will modify test.

Reply:
Thanks for getting some concrete examples checked in!

You may be having issues with the consent/org selection. As far as I know, only a few test organizations are added to the test database, so you may have to choose the "None of the Above" option or pick a known org.